cmd/influx/write.go

```diff
@@ -144,6 +149,30 @@ func (writeFlags *writeFlagsType) createLineReader(cmd *cobra.Command, args []st
 		}
 	}
 
+	// #18349 allow URL data sources, a simple alternative to `curl -f -s http://... | influx write ...`
+	if len(writeFlags.URLs) > 0 {
+		client := http.Client{Timeout: 5 * time.Minute}
```

Comment:
hmmm, is 5mins enough for a large write?

Reply:
changed to no timeout #18461